src/components/Button.js

```diff
@@ -4,13 +4,20 @@ import PropTypes from 'prop-types'
 import '../styles/Button.css'
 
 const Button = (props) => {
+  // var allowRegister = age > 18
+  console.log(props.wide)
```

Comment:
Please remove commented codes and delete console.log,  these things are only for debugging they should not make it to production.